dpcpp/CMakeLists.txt

```
@@ -8,6 +8,8 @@ set(GINKGO_DPCPP_VERSION ${GINKGO_DPCPP_VERSION} PARENT_SCOPE)
 
 find_package(MKL CONFIG REQUIRED HINTS "$ENV{MKLROOT}")
 set(GINKGO_MKL_ROOT "${MKL_ROOT}" PARENT_SCOPE)
+find_package(oneDPL REQUIRED HINTS "$ENV{DPL_ROOT}")
```

Should it be like MKL?
```suggestion
find_package(oneDPL CONFIG REQUIRED HINTS "$ENV{DPL_ROOT}")
```

---

I use the example from https://github.com/oneapi-src/oneDPL/tree/main/cmake#how-to-use-onedpl-package-from-cmake

---

They are not necessarily consistent inside Intel's documentation, and since there is no FindoneDPL.cmake module inside CMake (and I don't think there will be in the forseeable future), it doesn't really matter, except for slightly changing the error message.